ansible/templates/nomad.hcl.j2

```diff
@@ -12,9 +12,15 @@ advertise {
 
 bind_addr = "0.0.0.0"
 
+vault {
+  enabled = true
+  token = "master"
+  address = "http://127.0.0.1:8200"
+}
```

Comment:
These changes will be tested automatically in CD/CI. 

It might break current implementation with [vault_nomad_integration.yml](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/vault_nomad_integration.yml)